src/components/routes/Books.js

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
-import { v4 as uuidv4 } from 'uuid';
+// import { v4 as uuidv4 } from 'uuid';
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧